docs/api.adoc

```diff
@@ -579,33 +567,32 @@ When a renderer runs into an error during SSR, that renderer is torn down and a
+
+- react and reactDOM (`globals.*.js`),
 - packages from _node_modules_ (`vendors.*.js`),
 - dependency assets imported by the react components (aka. <<chunks#, chunks>>),
 - and finally, the <<entries#, entry assets>> themselves.
 
 *This causes some warmup time* when starting your app: a noticable delay before the first rendering shows up. This may be just a couple of seconds in total, but it may also take longer. It depends on the size and complexity of the _compiled_ assets involved. This will happen on every restart of the app (and every <<#renderers, renderer>> must be initialized, but they do this in parallel).
 
-But as long as the code runs in nashorn without errors, initialization happens only once (i.e. each asset is loaded once at most, on each renderer). After the warmup, the react apps are ready-to-run from nashorn memory, so repeated renderings after that (even with different props) are fast.
+But as long as the code runs without errors, initialization happens only once (i.e. each asset is loaded once at most, on each renderer). After the warmup, the react apps are ready-to-run from memory, so repeated renderings after that (even with different props) are fast.
 
-===== Improving warmup time for development
+==== Improving warmup time for development
 
-Since development can involve repeated app/server restarts, here are a couple of ways to improve initial loading time when developing large projects:
+Since development can involve repeated app/server restarts, here is one way to improve initial loading time when developing large projects:
 
-- Set `ssrLazyload = true` (see <<#ssrLazyload, below>>). This makes the engine start up and only load the bare minimum of dependency assets instead of preparing all of them at once. Each rendering will also only load/cache the dependencies it needs. The upside of this is that the first rendering (of a component that uses only a subset of the assets) is much faster since there is less to load. The downside is that there will still be assets that haven't been loaded into the engine yet, which will cause a delay at some other time when _they_ are first rendered.
-- Don't build the react components and dependencies with `BUILD_ENV = development`, but use `production` (which is the default - see <<#BUILD_ENV, below>>). Assets built with `development` are much more verbose, and this size difference - although functionally equal - actually makes a difference to nashorn (at the compile-to-bytecode stage).
+- Don't build the react components and dependencies with `BUILD_ENV = development`, but use `production` (which is the default - see <<#BUILD_ENV, below>>). Assets built with `development` are much more verbose, and this size difference - although functionally equal - actually makes a difference.
```

Comment:
> see <<#BUILD_ENV, below>>

This link is now broken since the section is removed.

Reply:
fixed